src/int/leading_zeros.rs

```diff
@@ -4,6 +4,7 @@
 // Compilers will insert the check for zero in cases where it is needed.
 
 /// Returns the number of leading binary zeros in `x`.
+#[doc(hidden)]
 pub fn usize_leading_zeros_default(x: usize) -> usize {
```

Comment:
Can this be `pub(crate)` instead of`#[doc(hidden)]`?

Reply:
It is being used by the testcrate (the CI does not have a RISCV target at the moment, so we want to make sure that they both work)